test/renders/form-semantic-formWithCustomFormatDate.html

```diff
@@ -24,7 +24,7 @@
         </thead>
         <tbody ref="datagrid-dataGrid-tbody">
           <tr ref="datagrid-dataGrid-row">
-            <td ref="datagrid-dataGrid" class="col-md-3">
+            <td ref="datagrid-dataGrid" class="col-md-">
```

Comment:
Is `col-md-` a class?  If not, isn't this testing that this is just broken?  This was added in a v4.13 rc..